go/enclave/main/enclave-test.json

```diff
@@ -18,6 +18,66 @@
     {
       "name": "EDG_TESTMODE",
       "value": "true"
-    }
+    },
+    { "fromHost": true, "name": "ENCLAVE_DB_EDGELESSDBHOST" },
```

Comment:
why not change the enclave.json as well?
Also, do we need both of these files?
Would be good to have a single enclave.json file